Monika After Story/game/script-topics.rpy

```diff
@@ -1074,6 +1074,47 @@ label monika_wine:
     m 2l "...Gosh, what am I talking about this for?"
     m "I don't condone underage drinking!"
     m 2a "I mean, I've never drank or anything, so...yeah."
+    m 3a "Speaking of which, I hear different parts of the world have different minimum drinking ages."
+    m "Would we both be able to have a drink where you're from?"
+    menu:
+        "Yes.":
+            m 1j "That's great!"
+            m 1k "Another thing to look forward to when we can finally be together!"
+            m 1l "I mean..."
+            m "Don't worry [player]. I'm not planning to become addicted to the stuff. I'm just excited is all."
```

Comment:
This sounds out of place. When people think about drinking, addiction is like the last thing on anyone's mind. I would take out most of the dialogue in this menu choice and maybe have monika be like "I cant wait to try alcohol" before being embarassed about that or something.